static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -80,7 +80,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
             };
           }}
           onClick={() => {
-            trackAnalytics('feedback_list.details_link.click', {organization});
```

Comment:
Does this still exist in analytic events? If so, we should remove it

Reply:
no it doesn't exist anywhere